src/export/export-chakra.ts

```diff
@@ -117,64 +123,96 @@ const processTextStyles = (
 
   // Replace the font family in each text style with the corresponding token
   Object.keys(updatedTextStyles).forEach((key) => {
-    const value = updatedTextStyles[key];
-    const fontFamily = value.fontFamily;
-    if (typeof fontFamily === 'string') {
-      const token = getFontFamilyToken(fontFamily);
-      if (token) {
-        value.fontFamily = token;
+    if (typeof updatedTextStyles[key] !== 'string') {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const value = updatedTextStyles[key];
+      const fontFamily = value.fontFamily;
+      if (typeof fontFamily === 'string') {
+        const token = getFontFamilyToken(fontFamily);
+        if (token) {
+          value.fontFamily = token;
+        }
       }
     }
   });
 
   return updatedTextStyles;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const reshapeDesignTokens = (tokens: any): ChakraTokens => {
+  const obj = tokens;
+
+  for (const key in obj) {
+    if (key === '$type') delete obj[key];
+
+    if (obj[key]?.$value) {
+      obj[key] = obj[key].$value;
+    }
+
+    if (Array.isArray(obj[key])) {
+      obj[key] = convertShadowsDesignTokenToCss(obj[key]);
+    }
+
+    if (obj[key] !== null && typeof obj[key] === 'object') {
+      reshapeDesignTokens(obj[key]);
+    }
+  }
+
+  return obj;
+};
+
 export default async function exportChakraFromTokens(
   tokens: Tokens,
   outputDir: string,
   figmaFileKey: string,
   versionDescription: string,
   fontFallbacks?: { [token: string]: string }
 ) {
+  const chakraTokens = reshapeDesignTokens(tokens);
+
   // Create a config for the templates by combining the design tokens with default Chakra values
   const chakra = {
-    breakpoints: tokens.breakpoints,
-    colours: tokens.colours,
-    gridStyles: tokens.gridStyles,
-    icons: await processIcons(tokens.icons),
+    breakpoints: chakraTokens.breakpoints,
+    colours: chakraTokens.colours,
+    gridStyles: chakraTokens.gridStyles,
+    icons: await processIcons(chakraTokens.icons),
     radii: {
       none: '0',
-      ...tokens.radii,
+      ...chakraTokens.radii,
       full: '9999px',
     },
     shadows: {
-      ...tokens.shadows,
+      ...chakraTokens.shadows,
```

Comment:
The colour values of the shadows are much less readable now, do we think that's a problem worth fixing?

![image](https://user-images.githubusercontent.com/890171/201814610-ff8ac7f1-8d5d-47d3-b4f7-ef43a4928234.png)

Reply:
Yes and no, [the spec](https://tr.designtokens.org/format/#color:~:text=The%20value%20MUST%20be%20a%20string%20containing%20a%20hex%20triplet/quartet%20including%20the%20preceding%20%23%20character.%20To%20support%20other%20color%20spaces%2C%20such%20as%20HSL%2C%20translation%20tools%20SHOULD%20convert%20color%20tokens%20to%20the%20equivalent%20value%20as%20needed.) currently says that colours have to be hex but that it likely to change as there is discussions about using other colour gamuts happening https://github.com/design-tokens/community-group/issues/137.
So I think it's probably best for us to leave it like this for now and then update to a more readable syntax when the spec changes.